src/foundry/client/ui/dialog.d.ts

```diff
@@ -187,7 +187,7 @@ declare namespace Dialog {
     /**
      * The name of the default button which should be triggered on Enter
      */
-    default: string;
+    default?: string;
```

Comment:
In order to make this work for people who use `exactOptionalPropertyTypes`, we should explicitly add `undefined` to the type.
```suggestion
    default?: string | undefined;
```